api-examples/tag-text.py

```diff
@@ -1,29 +1,62 @@
 import baseline as bl
 import argparse
+import json
 import os
+from baseline.utils import str2bool
 
 parser = argparse.ArgumentParser(description='Tag text with a model')
 parser.add_argument('--model', help='A tagger model with extended features', required=True, type=str)
 parser.add_argument('--text', help='raw value', type=str)
+parser.add_argument('--conll', help='is file type conll?', type=str2bool, default=False)
+parser.add_argument('--features', help='(optional) features in the format feature_name:index (column # in conll) or '
+                                       'just feature names (assumed sequential)', default=[], nargs='+')
 parser.add_argument('--backend', help='backend', default='tf')
 parser.add_argument('--remote', help='(optional) remote endpoint', type=str) # localhost:8500
 parser.add_argument('--name', help='(optional) signature name', type=str)
 parser.add_argument('--preproc', help='(optional) where to perform preprocessing', choices={'client', 'server'}, default='client')
-
+parser.add_argument('--grpc_feature_map', help='mapping between features and the fields in the grpc request,'
```

Comment:
This seems unnecessarily complex for an end user.  They shouldn’t have to define a JSON object on the command line

Reply:
for `features`, you know the indices of the json are 0,1 ... That is not true for this argument.